### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by Akatroj**

### CODE REVIEW

File: client/.env.development [hunk 1]
```diff
@@ -0,0 +1 @@
+VITE_BACKEND="localhost:4000"
```
Comment (dimamik) [inline client/.env.development:1]: don't need quotes there

File: client/src/screens/AdditionalQuestion/AdditionalQuestionScreen.tsx [hunk 2]
```diff
@@ -37,10 +36,10 @@ function AdditionalQuestionScreen() {
 
     await axios({
       method: 'post',
-      url: 'http://localhost:4000/add_question',
+      url: `${import.meta.env.VITE_BACKEND}/add_question`,
```
Comment (dimamik) [inline client/src/screens/AdditionalQuestion/AdditionalQuestionScreen.tsx:39]: process.env not working?

File: server/package.json [hunk 3]
```diff
@@ -8,23 +8,23 @@
...
     "node-json-db": "^1.5.0",
     "socket.io": "^4.5.0",
-    "ts-node": "^10.4.0"
+    "ts-node": "^10.4.0",
+    "@types/express": "^4.17.13",
+    "@types/lodash": "^4.14.182",
+    "@types/node": "^17.0.10",
+    "nodemon": "^2.0.15",
+    "prettier": "2.6.2",
+    "typescript": "^4.5.5"
```
Comment (dimamik) [inline server/package.json:27]: why?

**Review by dimamik — APPROVED** [review]
LGTM, Nothing blocking

File: client/src/screens/AdditionalQuestion/AdditionalQuestionScreen.tsx (hunk 2 above)
Comment (Akatroj) [reply client/src/screens/AdditionalQuestion/AdditionalQuestionScreen.tsx:39]: yes, https://vitejs.dev/guide/env-and-mode.html

File: server/package.json (hunk 3 above)
Comment (Akatroj) [reply server/package.json:27]: heroku is stupid and doesn't install dev dependencies, but we need tsc and types for build.

**Pull request merged**